test/config_tests/config_v1_test.py

```diff
@@ -69,7 +70,9 @@ def test_config_connection(self):
         # Default connection setting from xknx.yaml (auto:)
         self.assertEqual(
             TestConfig.xknx.connection_config,
-            ConnectionConfig(connection_type=ConnectionType.AUTOMATIC),
+            ConnectionConfig(
+                connection_type=ConnectionType.ROUTING, local_ip="192.168.111.201"
+            ),
```

Comment:
why is default connection Routing now?

Reply:
It is because I added all the values in the `xknx.yaml` file and the lastest value is used by the config reader class.
It will be better to use a dedicated `xknx.yaml` for test runners.

Reply:
Updated to go back to AUTO.